### DISCUSSION TITLE
How to animate text that move from right to left

### DISCUSSION THREAD

**Discussion by lawchin** [flet-dev/flet#1259] [Q&A] [answered]
### Question

How to animate text that move from right to left .
(coming out from right side of the screen and end to the left side of the screen) repeatedly every 10 seconds. just like marque behavior in html. 

And also how to animate picture just like a slide show.

see sample from this site:  https://suriagroup.com.my/

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
Maybe use offset? https://flet.dev/docs/guides/python/animations/#offset-animation

**Reply by lawchin** [reply]
ya.... this is exactly what I'm looking for.
